stan/math/rev/mat/fun/matrix_exp_multiply.hpp

```diff
@@ -388,23 +388,22 @@ matrix_exp_action(const Eigen::Matrix<Ta, N, N>& A,
 }
 
 /**
- * Return product of exp(At) and B, where A is a NxN double matrix,
- * B is a NxCb double matrix, and t is a double
+ * Wrapper of matrix_exp_action function for a more literal name
+ * @tparam Ta scalar type matrix A
  * @tparam N Rows and cols matrix A, also rows of matrix B
+ * @tparam Tb scalar type matrix B
  * @tparam Cb Columns matrix B
  * @param[in] A Matrix
  * @param[in] B Matrix
- * @param[in] t double
- * @return exponential of At multiplies B
+ * @return exponential of A multiplies B
  */
-template <int N, int Cb>
-inline Eigen::Matrix<double, N, Cb> matrix_exp_action(
-    const Eigen::Matrix<double, N, N>& A, const Eigen::Matrix<double, N, Cb>& B,
-    const double& t = 1.0) {
-  Eigen::Matrix<double, N, Cb> expAB;
-  matrix_exp_action_handler handle;
-  expAB = handle.action(A, B, t);
-  return expAB;
+template <typename Ta, int N, typename Tb, int Cb>
+inline typename boost::enable_if_c<boost::is_same<Ta, var>::value
```

Comment:
This would seem to fire for the `Ta, Tb == double` or `Tb == var`.   Some doc as to what's enabled here would be helpful.

Reply:
Removed `enable_if`.
```c++
template <typename Ta, typename Tb, int Cb>
inline Eigen::Matrix<typename stan::return_type<Ta, Tb>::type, -1, Cb>
matrix_exp_multiply(const Eigen::Matrix<Ta, -1, -1>& A,
                    const Eigen::Matrix<Tb, -1, Cb>& B) {
```